package.json

```diff
@@ -11,9 +11,11 @@
     "@fontsource/roboto-mono": "^4.4.2",
```

Comment:
Don't we need to update the `engines`? When I am `npm install`ing this branch, I see a warning that I should use `node v16.17.0` (it comes with `npm@8.15.0`)

Reply:
> Don't we need to update the engines? When I am npm installing this branch, I see a warning that I should use node v16.17.0 (it comes with npm@8.15.0)

Upgrading Node will mean updating some Dockerfiles, scripts, and the CI. The CI tests might take trial and error and a new runner, and for most devs there will a manual step or two to get their environment working again. I want to do it in another PR.

The 16.17 warning is safe to ignore; it comes from a Storybook CLI sub-sub-dependency. The Storybook dev workflow is unaffected:

- We don't use the feature that Storybook brought the package in for
- The package brings in another package for a feature that Storybook doesn't use either
- The sub-sub-sub dependency probably would work fine with Node 16.15 anyway, but asks for  >=16.17 for reasons unspecified by the author

…so our case that code never executes. **I'll push a commit** that pins a version so we don't have to see `EBADENGINE`:

```jsonc
"storybook": {
  "giget": "1.1.3" // Keeps execa <8 for unused feature
}
```

<details><summary>notes from researching this</summary>

These are the packages:

```
npm WARN EBADENGINE Unsupported engine {  
npm WARN EBADENGINE   package: 'execa@8.0.1',  
npm WARN EBADENGINE   required: { node: '>=16.17' },  
npm WARN EBADENGINE   current: { node: 'v16.15.0', npm: '8.5.5' }  
npm WARN EBADENGINE }  
npm WARN EBADENGINE Unsupported engine {  
npm WARN EBADENGINE   package: 'human-signals@5.0.0',  
npm WARN EBADENGINE   required: { node: '>=16.17.0' },  
npm WARN EBADENGINE   current: { node: 'v16.15.0', npm: '8.5.5' }  
npm WARN EBADENGINE }
```

Here's how we end up with it:

```
storybook cli   sandbox feature (that we don't use)
                  |
                  v
                giget   deps feature (that storybook doesn't use)
                            |
                            v
                          nypm   (gets updated by a bot)
                          |
                          v
                       execa -> human-signals (stricter versioning than required)
```

- storybook cli [switches from degit](https://github.com/storybookjs/storybook/commit/a9a5524d405a27c32c6df5c9bfe88fd10a4e961d) to giget ^1 [for a feature](https://github.com/storybookjs/storybook/blob/78c3f602ee2256f55ca269a9ad5c28bb9aea98e5/code/lib/cli/src/sandbox.ts#L31-L266) that [we don't use](https://storybook.js.org/docs/api/cli-options#sandbox) — Nov 18, 2022
- giget 1.2 adds nypm [for an optional feature](https://github.com/unjs/giget/blob/main/README.md?plain=1#L52). [storybook's feature that we don't use doesn't use that feature](https://github.com/storybookjs/storybook/blob/78c3f602ee2256f55ca269a9ad5c28bb9aea98e5/code/lib/cli/src/sandbox.ts#L203-L206). — Dec 23, 2023
- Meanwhile: 
  - execa 8 (and human-signals) have dropped Node 14, and in the process, specified a [stricter Node 16 version range](https://github.com/sindresorhus/execa/pull/569#discussion_r1284555820) (probably stricter than they need) — Aug 5, 2023
  - nypm 0.3.1 updates execa from ^7.2.0 to 8.0.1 [in a bot commit](https://github.com/unjs/nypm/commit/4d0b683eb0a6a91a8c0ce0600a8c3baa30aa457b) just to keep on top of things. — Aug 24, 2023
- Sometime later:
  - We update our dependencies. Since Storybook specifies giget ^1, it retrieves the latest version of giget, so we get the dependency warning coming from `giget -> nypm -> execa -> human-signals`.

</details>

Reply:
Thanks for the research! Looks fine to me :)